src/search/documents/paper.py

```diff
@@ -67,17 +70,68 @@ def should_remove_from_index(self, obj):
 
         return False
 
-    # Used specifically for "autocomplete" style suggest feature
-    def prepare_title_suggest(self, instance):
+    # Used specifically for "autocomplete" style suggest feature.
+    # Includes a bunch of phrases the user may search by.
+    def prepare_suggestion_phrases(self, instance):
+        phrases = []
+
+        # Variation of title which may be searched by users
+        if instance.title:
+            phrases.append(instance.title)
+            phrases.extend(instance.title.split())
+
+        if instance.doi:
+            phrases.append(instance.doi)
+
+        # Variation of journal name which may be searched by users
+        if instance.external_source:
+            journal_words = instance.external_source.split(" ")
+            phrases.append(instance.external_source)
+            phrases.extend(journal_words)
+
+        # Variation of OpenAlex keywords which may be searched by users
+        try:
+            oa_data = instance.open_alex_raw_json
+            keywords = [keyword_obj["keyword"] for keyword_obj in oa_data["keywords"]]
+            joined_kewords = " ".join(keywords)
+
+            phrases.append(joined_kewords)
+            phrases.extend(keywords)
+
+        except:
+            pass
+
+        # Variation of author names which may be searched by users
+        try:
+            authors_list = format_raw_authors(instance.raw_authors)
+            author_names_only = [
+                f"{author['first_name']} {author['last_name']}"
+                for author in authors_list
+                if author["first_name"] and author["last_name"]
+            ]
+            all_authors_as_str = ", ".join(author_names_only)
+
+            phrases.append(all_authors_as_str)
+            phrases.extend(author_names_only)
+        except:
+            pass
```

Comment:
Nit: Avoid bare exception:

```suggestion
        except Exception:
            pass
```